core.js

```diff
@@ -82,7 +83,7 @@ module.exports = function (xhr) {
       // For older servers, emulate JSON by encoding the request into an HTML-form.
       if (options.emulateJSON) {
           params.headers['content-type'] = 'application/x-www-form-urlencoded';
-          params.body = params.json ? {model: params.json} : {};
+          params.body = params.json ? {model: params.body || params.json} : {};
```

Comment:
Yep, you're right. 

The case this was attempting to address was if somebody passed a non-boolean truthy value to `json` in `options`. Except now I realize this is `json` in `params`, which will always be (true || undefined).

Making this change shortly.